TestPullRequest/src/testPackage/Calc.java

```diff
@@ -21,7 +21,7 @@ public Integer multiplication(int a, int b, int c) {
 	}
 
 	@Override
-	public void testInterface2() {
+	public void newName() {
```

Comment:
@LHommeDeBot: Can you please rename this method to something like 'newNamePlus'???? Contact me on 'https://www.example.com/foo/?bar=baz&inga=42&quux'.